src/components/Header.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const Header = () => (
+  <header style={{
```

Comment:
Kindly consider having these styles in a stylesheet(.css)  for ease of tracking and maintaining your application in future.